mvlearn/embed/kcca.py

```diff
@@ -124,6 +130,24 @@ class KCCA(BaseEmbed):
         {\sqrt{(\alpha'K_x^2\alpha+\kappa\alpha'K_x\alpha)
         \cdot (\beta'K_y^2\beta + \kappa\beta'K_y\beta)}}
 
+    Kernel matrices grow exponentially with the size of data. They not only
+    have to store :math:`n^2` elements, but also face the complexity of matrix
+    eigenvalue problems. In a Cholesky decomposition a positive definite
+    matrix A is decomposed to a lower triangular matrix :math:`L` :
+    :math:`A = LL'`.
+
+    The Incomplete Cholesky Decomposition (ICD) looks for a low rank
+    approximation of :math:`L` to reduce the cost of operations of the matrix
+    such that :math:`A $\approx$ $\tilde{L}$$\tilde{L}$'`. The algorithm skips
+    a column if its diagonal element is small. The diagonal elements to the
+    right of the column being updated are also updated. To select a column to
+    update, it finds the largest diagonal element and pivots the element to
+    the current diagonal by exchanging the corresponding rows and columns. The
+    algorithm ends when all diagonal elemnts are below a specified accuracy.
+
+    ICD with rank :math:`m` yields storage requirements of :math:`O(mn)`
```

Comment:
I can't readily think of a way to do ICD out of sample, unlike the full KCCA. I.e. you have to fit and transform on the same data always. I think this is fine but it should be mentioned in the description.

Reply:
okay